src/Altinn.App.Core/Models/ApplicationMetadata.cs

```diff
@@ -70,4 +70,10 @@ public ApplicationMetadata(string id)
     /// </summary>
     [System.Text.Json.Serialization.JsonExtensionData]
     public Dictionary<string, object>? UnmappedProperties { get; set; }
+
```

Comment:
This comment is for line 66 (not your code), because GH static code analysis triggers on it:

I would argue it's not breaking to change this `Exception` to something more specific, since that will still trigger a catch on `Exception` for callers. So if you're motivated to fix  this, even though it's not your code, please do 🙂  🙏